.github/workflows/jetbrains-update-plugin-platform.yml

```diff
@@ -5,23 +5,33 @@ on:
         # At 11:00 on every day-of-week from Monday through Friday.
         - cron: "0 11 * * 1-5"
 jobs:
-    update-backend-plugin-platform:
+    update-latest-backend-plugin:
         uses: ./.github/workflows/jetbrains-update-plugin-platform-template.yml
         with:
-            pluginName: JetBrains Backend Plugin
-            pluginId: backend-plugin
+            pluginName: JetBrains Backend Plugin (EAP)
+            pluginId: latest-backend-plugin
             xpath: "(/html/body/table[preceding::h2/text()='com.jetbrains.intellij.idea'][1]/tbody/tr/td[contains(text(),'-EAP-CANDIDATE-SNAPSHOT') and starts-with(text(),'MAJOR_VERSION_PLACEHOLDER')]/text())[1]"
             gradlePropertiesPath: components/ide/jetbrains/backend-plugin/gradle-latest.properties
         secrets:
             slackWebhook: ${{ secrets.IDE_SLACK_WEBHOOK }}
             roboquatRepoPat: ${{ secrets.ROBOQUAT_REPO_PAT }}
-    update-gateway-plugin-platform:
+    update-latest-gateway-plugin:
         uses: ./.github/workflows/jetbrains-update-plugin-platform-template.yml
         with:
-            pluginName: JetBrains Gateway Plugin
-            pluginId: gateway-plugin
+            pluginName: JetBrains Gateway Plugin (EAP)
+            pluginId: latest-gateway-plugin
             xpath: "(/html/body/table[preceding::h2/text()='com.jetbrains.gateway'][1]/tbody/tr/td[contains(text(),'-CUSTOM-SNAPSHOT') and starts-with(text(),'MAJOR_VERSION_PLACEHOLDER') and not(contains(text(),'-NIGHTLY'))]/text())[1]"
             gradlePropertiesPath: components/ide/jetbrains/gateway-plugin/gradle-latest.properties
         secrets:
             slackWebhook: ${{ secrets.IDE_SLACK_WEBHOOK }}
             roboquatRepoPat: ${{ secrets.ROBOQUAT_REPO_PAT }}
+    update-stable-gateway-plugin:
```

Comment:
Will this auto-merge too?

Reply:
It will. I think it's ok, considering it's executing the integration tests.

But we can change and set it to don't merge automatically. What do you think?

Reply:
I guess the beauty of this current workflow is that it updates EAP and avoids us manually doing any work, we trade of some risk in favour of getting time back. The integration tests only test connection so things can still break and we won't notice.

I am okay if EAP breaks occasionally but then if we break stable it can lead to incidents and time wasted which we could dedicate to test more before merging the PR (which is created by the other wofklow)

Reply:
> should we update stable gateway in [this other job](https://github.com/gitpod-io/gitpod/blob/main/.github/workflows/jetbrains-updates.yml)? together with backend and IDEs?

I don't think we need to.

> If we had stronger integration tests, I'd feel less concerned.

Let me set it to not auto-merge the PR of the Stable Gateway then.

Reply:
✅ Now we only auto-approve EAP Gateway Plugin updates.

https://github.com/gitpod-io/gitpod/blob/6c38d1bb45d675a59d9adf46a0fe3d6c47aa766c/.github/workflows/jetbrains-update-plugin-platform-template.yml#L104-L105